<launch>
  <node name="joint_controller" pkg="dynamic_controller" type="joint_controller" output="screen"></node>
  <include file="$(find potential_field_planner)/launch/joint_planner.launch"/>

  <rosparam command="load" file="$(find dynamic_controller)/config/dynamic_controller_params.yaml"/>

</launch>